New messages are added in Spanish

jorozcof opened the pull request:
Closes #19959

jorozcof committed:
New messages are added in Spanish

sessionLimitExceeded=Hay demasiadas sesiones.
federatedIdentityExistsMessage=El usuario con {0} {1} ya existe. Inicie sesi\u00F3n en la administraci\u00F3n de cuentas para vincular la cuenta.
federatedIdentityUnavailableMessage=El usuario {0} autenticado con el proveedor de identidad {1} no existe. P\u00F3ngase en contacto con su administrador.
identityProviderInvalidResponseMessage=Respuesta no v\u00E1lida del proveedor de identidad.
identityProviderMissingStateMessage=Falta el par\u00E1metro de estado en respuesta del proveedor de identidad.
alreadyLoggedIn=Ya se ha autentificado.

identity-provider-redirector=Con\u00E9ctese con otro proveedor de identidad
identity-provider-login-label=O inicia sesi\u00F3n con
idp-email-verification-display-name=Verificacion de email
idp-email-verification-help-text=Vincula tu cuenta validando tu correo electr\u00F3nico.
idp-username-password-form-display-name=Usuario y contrase\u00F1a
idp-username-password-form-help-text=Vincula tu cuenta iniciando sesi\u00F3n.

access-denied=Acceso denegado

logoutConfirmTitle=Saliendo de tu cuenta
logoutConfirmHeader=¿Quieres cerrar sesi\u00F3n?
doLogout=Cerrar sesi\u00F3n

rmartinc reviewed (changes requested):
Thanks for the PR  @jorozcof!

Just some minor comments because the messages mix second and third person, better use third person in all of them.

jorozcof commented on themes/src/main/resources-community/theme/base/login/messages/messages_es.properties:
```diff
@@ -192,10 +195,25 @@ identityProviderNotFoundMessage=No se encontr\u00F3 un proveedor de identidad.
+alreadyLoggedIn=Ya se ha autentificado.
 
+# Identity provider
+identity-provider-redirector=Con\u00E9ctese con otro proveedor de identidad
```
Yes, It sounds weird. I think is better `Inicie sesi\u00F3n con otro proveedor de identidad`. What do you thing @rmartinc ?

rmartinc replied:
Yep, it sounds good to me.

jorozcof committed:
Modification the messages in third person

https://github.com/keycloak/keycloak/issues/19959

rmartinc reviewed (approved):
Thanks @jorozcof! LGTM.

rmartinc commented:
Yes, no problem, you can add them in this PR. I'm going to set it to `request changes` to not merge it by mistake.

rmartinc reviewed (changes requested):
@jorozcof wants to add some more missing messages in Spanish.

jorozcof committed:
New translation messages are added

expiredActionMessage=Acci\u00F3n caducada. Contin\u00FAe con el inicio de sesi\u00F3n ahora.
invalidPasswordBlacklistedMessage=Contrase\u00F1a no v\u00E1lida: la contrase\u00F1a est\u00E1 en la lista negra.
configureBackupCodesMessage=Tienes que configurar c\u00F3digos de respaldo para activar su cuenta.
updateEmailMessage=Tienes que actualizar su direcci\u00F3n de correo electr\u00F3nico para activar su cuenta.
resetPasswordMessage=Tienes que cambiar tu contrase\u00F1a.
linkIdpMessage=Debe verificar su direcci\u00F3n de correo electr\u00F3nico para vincular su cuenta con {0}.

staleEmailVerificationLink=El enlace en el que hizo clic es un enlace obsoleto antiguo y ya no es v\u00E1lido. Tal vez ya hayas verificado tu correo electr\u00F3nico.
confirmAccountLinking=Confirme la vinculaci\u00F3n de la cuenta {0} del proveedor de identidad {1} con su cuenta.

confirmEmailAddressVerification=Confirme la validez de la direcci\u00F3n de correo electr\u00F3nico {0}.
confirmExecutionOfActions=Realice las siguientes acciones

proceedWithAction=&raquo; Haz click aqu\u00ED para proceder

requiredAction.CONFIGURE_TOTP=Configurar OTP
requiredAction.TERMS_AND_CONDITIONS=T\u00E9rminos y condiciones
requiredAction.UPDATE_PASSWORD=Actualizaci\u00F3n de contrase\u00F1a
requiredAction.UPDATE_PROFILE=Actualizaci\u00F3n del perfil
requiredAction.VERIFY_EMAIL=Verificar correo electr\u00F3nico
requiredAction.CONFIGURE_RECOVERY_AUTHN_CODES=Generar c\u00F3digos de recuperaci\u00F3n
requiredAction.webauthn-register-passwordless=Webauthn Registro sin contrase\u00F1a

pageNotFound=P\u00E1gina no encontrada
internalServerError=Se ha producido un error interno del servidor.

jorozcof commented:
> @jorozcof wants to add some more missing messages in Spanish.

@rmartinc Yes, I added new messages in Spanish. If you like, you can review in my last commit.

rmartinc reviewed (approved):
Thanks @jorozcof ! LGTM!

mposolda reviewed (approved):
@jorozcof @rmartinc Thanks for the PR and review! Merging based on @rmartinc review
